src/test/java/com/kaltura/client/test/tests/servicesTests/ottUserTests/ActivateTests.java

```diff
@@ -20,22 +20,25 @@
 
 public class ActivateTests extends BaseTest {
 
+    private Client client;
```

Comment:
Can it be moved into BaseTest or it is a problem for parallel running?

Reply:
I think it will create problems in parallel runs so I changed the baseTest client to be private